### PR TITLE
Set Field 

### PR DESCRIPTION
**Pull request by TechnologistTim**
This will populate a <u_location> field string based on the <location> field on the user's record without having to use GlideAjax. Go to the <u_location> dictionary and place this snippet in the Default Value field.

### CODE REVIEW

**Comment by sandeepd26** [comment]
Hi @TechnologistTim ,
Are you asking to put this condition as a flow action?

File: Flow Actions/Set Location/Set Field [hunk 1]
```diff
@@ -0,0 +1 @@
+javascript: var luser = new GlideRecord("sys_user"); luser.get("sys_id", gs.getUserID()); luser.location.u_installation_id.sys_id.toString();
```
Comment (SapphicFire) [inline Flow Actions/Set Location/Set Field:1]: This call type would not be suitable in a flow action. Additionally, you should use `getUniqueValue()` to get the sys_id string

File: Flow Actions/Set Location/Readme.ms [hunk 2]
```diff
@@ -0,0 +1 @@
+This will populate a <u_location> field string based on the <location> field on the user's record without having to use GlideAjax. Go to the <u_location> dictionary and place this snippet in the Default Value field.
```
Comment (SapphicFire) [inline Flow Actions/Set Location/Readme.ms:1]: Invalid file type, `u_installation_id` from the script is not documented, and this readme does not describe much at all

**Review by SapphicFire — CHANGES_REQUESTED** [review]
Thank you for your contribution. However, it doesn’t comply with our contributing guidelines. As a reminder, the general requirements (as outlined in the [CONTRIBUTING.md file](https://github.com/ServiceNowDevProgram/code-snippets/blob/main/CONTRIBUTING.md)) are the following: follow the folder+subfolder guidelines and include a readme.md file explaining what the code snippet does. Review your contribution against the guidelines and make the necessary adjustments. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

File: Flow Actions/Set Location/Readme.ms (hunk 2 above)
Comment (TechnologistTim) [reply Flow Actions/Set Location/Readme.ms:1]: Sorry, I looked at another readme from someone else and though I followed what they did. I'll add more detail. 

As for the filetype, I followed the contribution file but for a newb like me, it doesn't provide the level of detail I need. I simply added a file as it states and typed away.

As for the field. I'm going to add in the Readme this applies to a reference field that is added. "u_installation" being the name of my particular field. The <> was an attempt to say "whatever you want to name it".

File: Flow Actions/Set Location/Set Field (hunk 1 above)
Comment (TechnologistTim) [reply Flow Actions/Set Location/Set Field:1]: It works flawlessly on our production instance as is. I did mean to add this to Dynamic Filter though, not Flow Action.

**Pull request closed without merging**